runtime/oti/jvminit.h

```diff
@@ -419,6 +419,8 @@ enum INIT_STAGE {
 #define VMOPT_XXNOOPENJ9COMMANDLINEENV "-XX:-OpenJ9CommandLineEnv"
 #define VMOPT_XXCPULOADCOMPATIBILITY "-XX:+CpuLoadCompatibility"
 #define VMOPT_XXNOCPULOADCOMPATIBILITY "-XX:-CpuLoadCompatibility"
+#define VMOPT_XXENABLEEXTENDEDHCR "-XX:+enableExtendedHCR"
+#define VMOPT_XXDISABLEEXTENDEDHCR "-XX:-enableExtendedHCR"
```

Comment:
Are the options case sensitive? All other XX options have the first letter capitalized.